notebooks/233-encodec-audio-compression/233-encodec-audio-compression.ipynb

```diff
@@ -29,7 +48,32 @@
     }
    ],
    "source": [
-    "!pip install encodec"
+    "!pip install -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82279f60-a018-4ea8-9b0b-77f0f40903cf",
+   "metadata": {},
+   "source": [
+    "## Instantiate audio compression pipeline\n",
+    "\n",
+    "[Codecs](https://en.wikipedia.org/wiki/Codec), which act as encoders and decoders for streams of data, help power most of the audio compression people currently use online. Some examples of commonly used codecs include MP3, Opus, and EVS. Classic codecs like these decompose the signal between different frequencies and encode as efficiently as possible. Most classic codecs leverage human hearing knowledge (psychoacoustics) but have a finite or given set of handcrafted ways to efficiently code and decode the file. EnCodec, a neural network that is trained end to end to reconstruct the input signal, was introduced as attempt to overcome this limitation. It consits of 3 parts:\n",
+    "\n",
+    "* The **encoder**, which takes the uncompressed data and transforms it into a higher dimensional and lower frame rate representation.\n",
+    "\n",
+    "* The **quantizer**, which compresses this representation to the target size. This compressed representation is what is stored on disk or will be send through the network.\n",
+    "\n",
+    "* The **decoder** is the final step. It turns the compressed signal back into a waveform that is as similar as possible to the original. The key to lossy compression is to identify changes that will not be perceivable by humans, as perfect reconstruction is impossible at low bit rates.\n",
+    "\n",
+    "![encodec_compression](https://scontent-lhr8-1.xx.fbcdn.net/v/t39.2365-6/312631767_820469602333729_4265486792384109107_n.png?_nc_cat=108&ccb=1-7&_nc_sid=ad8a9d&_nc_ohc=92B-ND5hEs0AX_ALCwD&_nc_ht=scontent-lhr8-1.xx&oh=00_AfDN4ggNCYyPVGXEZKG6F8rZ_TP0_NFhUcG7Z2n8FdqEtg&oe=64016841)\n",
+    "\n",
+    "The authors provide our two multi-bandwidth models:\n",
+    "* `encodec_model_24khz` - a causal model operating at 24 kHz on monophonic audio trained on a variety of audio data.\n",
+    "* `encodec_model_48khz` - a non-causal model operationg at 48 kHz on stereophonic audio trained on music-only data.\n",
+    "\n",
+    "in this tutorial, we will use `encodec_model_24khz` as example, but the same actions are also applicable to `encodec_model_48khz` model as well.\n",
+    "For starting work with model, we need to instantiate model class using `EncodecModel.encodec_model_24khz()` and select required compression bandwidth among available: 1.5, 3, 6, 12 or 24 kbps for 24 kHz model and 3, 6, 12 and 24 kbps for 48 kHz model. We will use 6 kbs bendwidth."
```

Comment:
To start working with this model, we need to instantiate model class using `EncodecModel.encodec_model_24khz()` and select required compression bandwidth among available: 1.5, 3, 6, 12 or 24 kbps for 24 kHz model and 3, 6, 12 and 24 kbps for 48 kHz model. We will use 6 kbs bandwidth.